polartools/diffraction.py

```diff
@@ -93,7 +93,7 @@ def fit_peak(xdata, ydata, model="Gaussian", scan=None, output=False):
             )
         plt.plot(xdata, ydata)
         plt.plot(xdata, fit.best_fit)
-        plt.show()
+        plt.show(block=True)
```

Comment:
You can leave it as is for now, but you should test using `plt.ion()` (I think it can go right after you import `plt`). With `plt.ion()` you don't need to do `plt.show()`.